src/System.Runtime.WindowsRuntime/src/System/IO/BufferedStreamWrapper.cs

```diff
@@ -0,0 +1,208 @@
+        {
+            get
+            {
+                EnsureNotClosed();
+                return _bufferedStream.Position;
+            }
+            set
+            {
+                EnsureNotClosed();
+                _bufferedStream.Position = value;
+            }
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            try
+            {
+                if (disposing && _bufferedStream != null)
+                {
+                    _bufferedStream.Dispose();
+                }
+            }
+            finally
+            {
+                _bufferedStream = null;
+                _stream = null;
+            }
+        }
+
+        public override void Flush()
+        {
+            EnsureNotClosed();
+            _bufferedStream.Flush();
+        }
+
+        public override Task FlushAsync(CancellationToken cancellationToken) 
+        {
+             EnsureNotClosed();
+             return _bufferedStream.FlushAsync();
+        }
+
+        public override IAsyncResult BeginRead(Byte[] buffer, int offset, int count, AsyncCallback callback, Object state)
+        {
+            EnsureNotClosed();
+            return _bufferedStream.BeginRead(buffer, offset, count, callback, state);
+        }
+
+        public override int EndRead(IAsyncResult asyncResult)
+        {
+            EnsureNotClosed();
+            return _bufferedStream.EndRead(asyncResult);
+        }
+
+        public override int Read(byte[] array, int offset, int count)
+        {
+            EnsureNotClosed();
+            return _bufferedStream.Read(array, offset, count);
+        }
+
+        public override Task<int> ReadAsync(Byte[] buffer, int offset, int count, CancellationToken cancellationToken)
+        {
+            EnsureNotClosed();
+            return _bufferedStream.ReadAsync(buffer, offset, count, cancellationToken);
+        }
+
+        public override int ReadByte()
+        {
+            EnsureNotClosed();
+            return _bufferedStream.ReadByte();
+        }
+
+        public override void Write(Byte[] array, int offset, int count)
+        {
+            EnsureNotClosed();
+            _bufferedStream.Write(array, offset, count);
+        }
+
+        public override IAsyncResult BeginWrite(Byte[] buffer, int offset, int count, AsyncCallback callback, Object state)
+        {
+            EnsureNotClosed();
+            return _bufferedStream.BeginWrite(buffer, offset, count, callback, state);
+        }
+
+        public override void EndWrite(IAsyncResult asyncResult)
+        {
+           EnsureNotClosed();
+            _bufferedStream.EndWrite(asyncResult);
+        }
+
+        public override Task WriteAsync(Byte[] buffer, int offset, int count, CancellationToken cancellationToken)
+        {
+            EnsureNotClosed();
+            return _bufferedStream.WriteAsync(buffer, offset, count, cancellationToken);
+        }
+
+        public override void WriteByte(Byte value)
```

Comment:
Thanks for this. also created an issue for some other files under System.Runtime.WindowsRuntime doesn't use this code style.

https://github.com/dotnet/corefx/issues/12384